packages/core/src/actors.ts

```diff
@@ -346,73 +228,82 @@ export function createPromiseBehavior<T, TEvent extends EventObject>(
   return behavior;
 }
 
-export function createObservableBehavior<
-  T extends EventObject,
-  TEvent extends EventObject
->(lazyObservable: Lazy<Subscribable<T>>): Behavior<TEvent, T | undefined> {
-  const parent = CapturedState.current?.actorRef;
+export function fromObservable<T, TEvent extends EventObject>(
+  lazyObservable: Lazy<Subscribable<T>>
+): Behavior<TEvent, T | undefined> {
```

Comment:
so... in a way, this should be:
```suggestion
): Behavior<never, T | undefined> {
```
or with the latest changes:
```suggestion
): Behavior<{ type: string }, T | undefined> {
```
With this design, an observable is really closer to a promise than a machine/callback (they can send events directly to the parent)